### PR TITLE
Daily PR April 5

### PR DESCRIPTION
**Pull request by pe7r**
Work with Delete selected, styles, fixed bugs.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -54,6 +54,28 @@ class App extends Component {
 		})
 	}
 
+	deleteSelected = () => {
+	const { tasks, checkedTasks } = this.state;
+
+	const deletedItems = []
+	const checkedItems = []
+
+	tasks.forEach(task => {
```
Comment (ruslan-palahin) [inline src/App.js:63]: May it be an option?
```

this.setState({
	   tasks: tasks.filter(item => !checkedTasks.include(item)),
	   checkedTasks: [],
	 })
```

File: src/App.js [hunk 2]
```diff
@@ -145,10 +167,13 @@ class App extends Component {
 		},() => {
 		if (this.state.checkRules === 'check') {
 			this.checkAll()
+			this.setState({ checkRules: '' })
 		} else if (this.state.checkRules === 'uncheck') {
 			this.uncheckAll()
+			this.setState({ checkRules: '' })
 		} else if (this.state.checkRules === 'delete') {
 			this.deleteSelected()
+			this.setState({ checkRules: '' })
```
Comment (ruslan-palahin) [inline src/App.js]: Move it under "if" statement to use only once.

**Review by ruslan-palahin — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 2 above)
Comment (pe7r) [reply src/App.js]: Changed.

File: src/App.js (hunk 1 above)
Comment (pe7r) [reply src/App.js:63]: I think I need to left it, cause I need to use forEach for newTasks array.

**Pull request merged**